requirements.txt

```diff
@@ -13,3 +17,5 @@ boto3>=1.4.4
 -e git+https://github.com/ckan/ckanext-issues.git#egg=ckanext-issues
 
 -e git+https://github.com/OpenUpSA/ckanext-gdoc.git#egg=ckanext-gdoc
+
+-e git+https://github.com/ckan/ckanext-harvest.git#egg=ckanext-harvest
```

Comment:
@phillipahereza We are using this harvester actually - https://github.com/CodeForAfricaLabs/ckanext-harvest . It allows to scrape socrata sites as well.

Also, we should update README.md on this.

Reply:
@DavidLemayian this harvest plugin doesn't work, I think it needs to be updated.